ngx-tools/jwt/src/jwt-token-managment/utilities/retry-with-escalation.ts

```diff
@@ -27,8 +27,8 @@ import {
   take,
 } from 'rxjs/operators';
 
-import * as JwtActions from '../actions';
-import { ClaimMap } from '../claim-map';
+import * as JwtActions from './../actions';
+import { ClaimMap } from './../claim-map';
```

Comment:
Just for you @deanterm..

Reply:
![image](https://user-images.githubusercontent.com/41060343/61893417-06b3de00-aedc-11e9-80bf-a0533d410128.png)